tests/test_base.py

```diff
@@ -183,7 +187,7 @@ class AbstractPositionTest(AbstractVectorTest):
     """Test :class:`coordinax.AbstractPosition`."""
 
     @pytest.fixture(scope="class")
-    def vector(self) -> AbstractPosition:  # noqa: PT004
+    def vector(self) -> AbstractPosition:
```

Comment:
This rule is deprecated: https://docs.astral.sh/ruff/rules/pytest-missing-fixture-name-underscore/